tasks/rtloader.py

```diff
@@ -160,7 +161,8 @@ def generate_doc(ctx):
 
     # doxygen puts both errors and warnings in stderr
     result = ctx.run(
-        "doxygen '{0}/doxygen/Doxyfile' 2>'{0}/doxygen/errors.log'".format(rtloader_path), warn=True  # noqa: FS002
```

Comment:
FS002 is a lint about using `.format`, so I presume the use here was intentional. Is it okay to remove it? If so, let's remove the lint silencer.

Reply:
I think it's ok, but you're right since this is probably intentional. I added it back and used the `# noqa: UP032` which is the ruff rule for this.